base/dates/conversions.jl

```diff
@@ -19,9 +19,20 @@ function unix2datetime(x)
 end
 # Returns unix seconds since 1970-01-01T00:00:00
 datetime2unix(dt::DateTime) = (value(dt) - UNIXEPOCH)/1000.0
+
+_tzoffset = 0
+function settzoffset()
+    t = floor(time())
+    tm = Libc.TmStruct(t)
+    dtutc = unix2datetime(t)
+    dtlocal = DateTime(tm.year+1900,tm.month+1,tm.mday,tm.hour,tm.min,tm.sec)
+    _tzoffset = Second(dtlocal - dtutc).value
+end
+settzoffset() #set it on load
+tzoffset() = _tzoffset
```

Comment:
Yes it's broken - the setter needs to be in `__init__` - cheers.
if I use: `_tzoffset = float(Second(dtlocal - dtutc).value) I wasn't sure about what type to check for in the line in now:

    t = time() + _tzoffset::Int

AbstractFloat? I preferred a concrete type, is there a type alias for the type that `float` returns?